### PR TITLE
MEN-6861: Code fixes

### PR DESCRIPTION
**Pull request by lluiscampos**

### CODE REVIEW

File: cli/cli.go [hunk 1]
```diff
@@ -79,6 +79,12 @@ func SetupCLI(args []string) error {
 				Usage:       "Dumps rootfs to stdout.",
 				Action:      runOptions.DumpSnapshot,
 				Flags: []cli.Flag{
+					&cli.StringFlag{
+						Name:    "data",
+						Aliases: []string{"d"},
+						Usage:   "Mender state data `DIR`ECTORY path.",
```
Comment (ghost) [inline cli/cli.go:85]: Weird quoting, is it intentional?

**Review by ghost — APPROVED** [review]

File: cli/cli.go (hunk 1 above)
Comment (lluiscampos) [reply cli/cli.go:85]: I took it like that from the [mender](https://github.com/mendersoftware/mender/blob/5969d1b5727d9a89bf57cfcd81a1e2d50072105f/cli/cli.go#L228) code base, and we also have the same in [mender-setup](https://github.com/mendersoftware/mender-setup/blob/c9a735681f624c944ea2fffcf9e4e528cc43606d/cli/cli.go#L84) code.

Although I cannot point you out to docs or code to prove it, it is used somehow by the library to put the quoted text as the example argument in the help text. In other words, the help will say `--data DIR, d DIR` based on that quoted `DIR` from the Usage text.

On my terminal:
```
   --data DIR, -d DIR             Mender state data DIRECTORY path. (default:
                                  "/var/lib/mender")
```

File: cli/snapshot.go [hunk 2]
```diff
@@ -86,7 +86,18 @@ func newWatchDog() *watchDog {
...
+	// Ensure we don't write logs to the filesystem
 	log.SetOutput(os.Stderr)
+	if ctx.Bool("quiet") {
+		log.SetLevel(log.ErrorLevel)
+	} else {
+		if lvl, err := log.ParseLevel(ctx.String("log-level")); err == nil {
+			log.SetLevel(lvl)
+		} else {
+			log.Warnf(
+				"Failed to parse set log level '%s'.", ctx.String("log-level"))
```
Comment (oleorhagen) [inline cli/snapshot.go:98]: Hmm, is this right? Does `ctx.String(log-level)` have a default?

Comment (oleorhagen) [reply cli/snapshot.go:98]: I see previously it was set to error by default

**Review by oleorhagen — APPROVED** [review]
Nice job!

**Pull request merged**